src/kontrol/prove.py

```diff
@@ -599,14 +602,19 @@ def _method_to_cfg(
         init_node_id = init_node.id
 
     final_cterm = _final_cterm(
-        empty_config, program, failing=method.is_testfail, is_test=method.is_test, is_setup=method.is_setup, hevm=hevm
+        empty_config,
+        program,
+        failing=method.is_testfail,
+        is_test=method.is_test,
+        is_setup=method.is_setup,
+        hevm=hevm,
```

Comment:
I don't see how this is different... am I missing something? The `,` at the end causes the whole thing to no longer be one-line friendly?

Reply:
Hmm, seems like that's the case. I don't remember adding the comma on purpose.